### ISSUE TITLE
Could you tell me detailed conditions of early stopping?

### ISSUE THREAD

**Issue by ken-take-it-so-so** [zalandoresearch/pytorch-ts#114]
To @kashif,  
I'm very glad to read "Autoregressive Denoising Diffusion Models for Multivariate Probabilistic Time Series Forecasting" . I'm sure that this paper reveals the power of diffusion models. However, I tried to decide the number of epochs using early stopping, can't reproduce it. Could you tell me detailed conditions of early stopping and how to make a validation set?
I would appreciate your cooperation.
